ush/forecast_postdet.sh

```diff
@@ -435,47 +435,48 @@ FV3_GFS_postdet(){
           $NLN $FIX_AM/qr_acr_qs.dat  qr_acr_qs.dat
         fi
 
-        if [ $inistep = 'cold' ]; then
-          echo "Not making links of output for mediator cold start" 
-        else 
-	        if [ $QUILTING = ".true." -a $OUTPUT_GRID = "gaussian_grid" ]; then
-	          fhr=$FHMIN
-	          while [ $fhr -le $FHMAX ]; do
-	            FH3=$(printf %03i $fhr)
-                    FH2=$(printf %02i $fhr)
-	            atmi=atmf${FH3}.$OUTPUT_FILE
-	            sfci=sfcf${FH3}.$OUTPUT_FILE
-	            logi=logf${FH3}
-                    pgbi=GFSPRS.GrbF${FH2}
-                    flxi=GFSFLX.GrbF${FH2}
-	            atmo=$memdir/${CDUMP}.t${cyc}z.atmf${FH3}.$OUTPUT_FILE
-	            sfco=$memdir/${CDUMP}.t${cyc}z.sfcf${FH3}.$OUTPUT_FILE
-	            logo=$memdir/${CDUMP}.t${cyc}z.logf${FH3}.$OUTPUT_FILE
-                    pgbo=$memdir/${CDUMP}.t${cyc}z.master.grb2f${FH3}
-                    flxo=$memdir/${CDUMP}.t${cyc}z.sfluxgrbf${FH3}.grib2
-	            eval $NLN $atmo $atmi
-	            eval $NLN $sfco $sfci
-	            eval $NLN $logo $logi
-                    if [ $WRITE_DOPOST = ".true." ]; then
-                      eval $NLN $pgbo $pgbi
-                      eval $NLN $flxo $flxi
-                    fi
-	            FHINC=$FHOUT
-	            if [ $FHMAX_HF -gt 0 -a $FHOUT_HF -gt 0 -a $fhr -lt $FHMAX_HF ]; then
-	              FHINC=$FHOUT_HF
-	            fi
-	            fhr=$((fhr+FHINC))
-	          done
-	        else
-	          for n in $(seq 1 $ntiles); do
-	            eval $NLN nggps2d.tile${n}.nc       $memdir/nggps2d.tile${n}.nc
-	            eval $NLN nggps3d.tile${n}.nc       $memdir/nggps3d.tile${n}.nc
-	            eval $NLN grid_spec.tile${n}.nc     $memdir/grid_spec.tile${n}.nc
-	            eval $NLN atmos_static.tile${n}.nc  $memdir/atmos_static.tile${n}.nc
-	            eval $NLN atmos_4xdaily.tile${n}.nc $memdir/atmos_4xdaily.tile${n}.nc
-	          done
-	        fi
-        fi
+        affix="nc"
+        if [ "$OUTPUT_FILE" = "nemsio" ]; then
+          affix="nemsio"
+        fi 
+
+	if [ $QUILTING = ".true." -a $OUTPUT_GRID = "gaussian_grid" ]; then
+	  fhr=$FHMIN
+	  while [ $fhr -le $FHMAX ]; do
+	    FH3=$(printf %03i $fhr)
+            FH2=$(printf %02i $fhr)
```

Comment:
Indentation doesn't match surrounding

Reply:
These were all issues of a mix of tabs and spaces being used, they line up when looking at the routines in vi but not when looking at it in GitHub.  All tabs were replaced by spaces in this routine.  All ush/forecast_postdet.sh indentation issues should be fixed.

Reply:
At some point, all spaces will need to be replaced with tabs in all g-w bash scripts, but that is not something you need to worry about. This is fine for now.